### PR TITLE
Add time zone support for timeseries 

### PR DESCRIPTION
**Pull request by lpereiracgn**
* This PR adds support for time zones in Timeseries, the user can select the timezone using the Grafana time range options.
demo-tz

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cognite/cognite-grafana-datasource",
-  "version": "4.1.3",
+  "version": "4.1.4",
```
Comment (polomani) [inline package.json:3]: @lpereiracgn  https://semver.org/#summary

**Review by polomani — COMMENTED** [review]
PR looks good (description and version number has to be fixed)
We can merge it after the time series feature is GA

TODO before merge:
- [] delete beta header
- fix version and PR/commit description

File: src/plugin.json [hunk 2]
```diff
@@ -65,6 +65,10 @@
         {
           "name": "x-cdp-app",
           "content": "Cognite Grafana Datasource"
+        },
+        {
+          "name": "cdf-version",
+          "content": "beta"
```
Comment (matiascognite) [inline src/plugin.json:71]: No longer needed :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,12 @@
 This article documents the ongoing improvements we're making to the **Cognite
 Data Source for Grafana**.
 
+## 4.1.3 - June 17th, 2024
```
Comment (polomani) [inline CHANGELOG.md:6]:
```suggestion
## 4.2.0 - June 17th, 2024
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
...
+
+### Features
+
+- Adds support for time zones in Timeseries, the user can select the timezone using the Grafana time range options.
```
Comment (polomani) [inline CHANGELOG.md:10]:
```suggestion
- Adds time zone support for for aggregated time series, the user can select the timezone using the Grafana time range options.
```

File: package.json (hunk 1 above)
Comment (polomani) [inline package.json:3]: bump minor version, not patch, follow the semantic release guidelines

**Review by polomani — APPROVED** [review]

**Pull request merged**